### DISCUSSION TITLE
Embedded Video Titles Not Appearing on Quarto Website

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#4880] [Q&A] [answered]
I am building a course website that has embedded videos for each module. When I insert the video code I include a title like this: 

`{{< video https://www.youtube.com/embed/wo9vZccmqwc title="Downloading Data from the World Bank" >}}`

(Note that the video I am just using the CERN video as a placeholder here). 

But [as you can see](https://dataviz-gwu.rocks/modules/module-1.1.html#downloading-world-bank-data-into-a-flat-file), the title does not appear anywhere on the video when I render the site. [Here](https://github.com/eteitelbaum/dataviz-summer-2023/edit/main/modules/module-1.1.qmd) is the full code for the page. 

What am I missing?

**Comment by cderv** [comment]
What are you expecting exactly for the title ?

You have at https://github.com/eteitelbaum/dataviz-summer-2023/blob/e5ad6c54cded454d8aae6d9d504e8b8cddc574c5/modules/module-1.1.qmd#L80
````
{{< video https://youtu.be/ud36szsO9a4 title='Reshaping Data With `pivot_longer()`' >}}
````

and the HTML page at https://dataviz-gwu.rocks/modules/module-1.1#reshaping-data has
````html
<iframe data-external="1" src="https://www.youtube.com/embed/u4AXi8A1Bqo" title="Reshaping Data With `pivot_longer()`" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
````

As you can see the `title` attribute is included, which is the recommended attributes for accessibility: https://dequeuniversity.com/tips/provide-iframe-titles

Are you expecting something else from the `title` ?

**Reply by eteitelbaum** [reply, marked as answer]
Hi is there a reason you are reopening this thread? I asked the question over three years ago and am getting spam messages attached to it now.

**Reply by mcanouil** [reply]
@eteitelbaum closing discussion does not block spammers, it only make the discussion not searchable by default on GitHub search (GitHub sets `is:open`).
If you don't want notifications, then the solution is to unsubscribe via the big button on the right of the thread.

**Reply by eteitelbaum** [reply]
@mcanouil I am not sure I understand this behavior. You are answering the question almost three years after it was asked and then reopening it after the issue has been resolved. Yes I will unsubscribe!

**Reply by mcanouil** [reply]
I did not do anything beside letting the discussion open after you first closed it, as said the team do not close discussions for the reason mentioned earlier.
You got notifications because of the two accounts that I marked as spams (already reported to GitHub) and not from any actions the team or I did.

<img width="338" height="225" alt="discussion activity screenshot" src="https://github.com/user-attachments/assets/a0f90d83-f144-4e05-8887-ef50391f31d2" />

**Reply by eteitelbaum** [reply]
@mcanouil Right. It was another team member, @cderv, who responded almost three years later and now you are reopening it. His response suggested that it was a non-issue and I was able to resolve it so I closed the issue. Do you see a strong reason for keeping it open and searchable?

**Reply by mcanouil** [reply]
yes, but it was already 6 months ago.

Besides search, a discussion being closed or opened has absolutely no effects, i.e., anyone including scammers can still post comments.
So keeping it open help users searching with GitHub find this.


edit: btw, every time you participate in a thread (PRs, Issues, and Discussions) on GitHub you subscribe to it and get notifications baed on your settings. -> unsubscribe without replying after

**Comment by jamesdaniels2678-cmd** [comment]
I was searching for a working SAB spawner script and came across this page:
 — it looks quite useful and seems to have updated [Roblox script]( https://gag-scripts.com/script/sab-spawner-script-by-xware) features for players looking for new tools.

**Comment by colinmiranda3789-hub** [comment]
Estas[ letras bonitas graffiti tienen un diseño ](https://letrasbonitass.com.mx/)urbano y creativo que destaca en cualquier proyecto artístico. También encontré más estilos interesantes en
